### PR TITLE
Update visualization.md(단어 수정)

### PR DESCRIPTION
**Pull request by fliklab**
**<단어 수정>**
- 플롯팅, 플롯 →그래프
- 디폴트 값 → 기본값
- 인수 → 인자
- 레이어 → 층
- 모양(shape) → 형태

**보류**
_전체적인 내용 수정해야 하거나 논의가 필요한 단어._ 
- 학습측정항목값
- 학습 손실값(training loss values)
- 학습 정확성 값(training accuracy values)
- 검증 손실 값(validation loss values)
- 검증 정확성 값(validation accuracy values)
- 플롯(전체적인 내용이 함께 수정되어야 하는 경우)

**그대로 유지**
- 에폭, 클러스터

### CODE REVIEW

**Comment by fuzzythecat** [comment]
정확성 값은 정확도로 바꿔보는 것은 어떨까요? accuracy value에서 value는 불필요해 보입니다.

File: sources/visualization.md [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 
-아래 예시는 모델의 그래프를 플롯팅하고 그 결과를 파일로 저장합니다:
+아래 예시는 모델의 그래프를 그려주고 그 결과를 파일로 저장합니다:
 ```python
 from keras.utils import plot_model
 plot_model(model, to_file='model.png')
 ```
 
-`plot_model`는 네가지 인수를 전달받습니다:
+`plot_model`는 네가지 인자를 전달받습니다:
```
Comment (Denny-Hwang) [inline sources/visualization.md]: 는->은으로 하는게 좋을 것 같습니다.

Comment (fliklab) [reply sources/visualization.md]: 단어 수정단계라서

> 정확성 값은 정확도로 바꿔보는 것은 어떨까요? accuracy value에서 value는 불필요해 보입니다.

해당단어가 문단 내용을 전체적으로 수정할 필요가 있어서 별도 PR로 생성하였는데 다음 PR에 commit해도 괜찮을지요?

File: sources/visualization.md [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 
-- `show_shapes`(디폴트값은 False)는 결과의 모양을 그래프에 나타낼 것인지 조정합니다.
-- `show_layer_names`(디폴트값은 True)는 레이어의 이름을 그래프에 나타낼 것인지 조정합니다.
-- `expand_nested`(디폴트값은 False)는 중첩된 모델을 그래프상에서 클러스터로 확장할 것인지 조정합니다.
-- `dpi`(디폴트값은 96)는 이미지 dpi를 조정합니다.
+- `show_shapes`(기본값은 False)는 결과의 형태을 그래프에 나타낼 것인지 조정합니다.
```
Comment (Denny-Hwang) [inline sources/visualization.md]: '조정'이라는 단어를 설정으로 바꾸면 어떨까요?
원문은 'control'이긴한데 조정은 'tuning'과 헷갈리수도 있을 것 같아서요.

File: sources/visualization.md (hunk 1 above)
Comment (Denny-Hwang) [reply sources/visualization.md]: 넵!!
